src/pages/blog/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import MarketNews from './_markets-news'
+import Layout from 'components/layout/layout'
+import { WithIntl } from 'components/localization'
+
+const DerivBlog = () => {
```

Comment:
Could we create this for QA testing on a different route for the mean time to avoid conflicts? Maybe `/blog/weekly-reports`? We can omit move this index.js file there.